logical/framework/path.go

```diff
@@ -23,6 +24,12 @@ func OptionalParamRegex(name string) string {
 	return fmt.Sprintf("(/(?P<%s>.+))?", name)
 }
 
+// Helper which returns a regex string for capturing an entire endpoint path
+// as the given name.
+func MatchAllRegex(name string) string {
```

Comment:
This doesn't appear to be in use, nor have test coverage.

Reply:
It will be used once the changes to the KV plugin land. I'll add a test for it.

Reply:
Gotcha, thank you!